### DISCUSSION TITLE
Installing File Browser  - ERROR "1@64" Unknown failure occured.

### DISCUSSION THREAD

**Discussion by timdonovanuk** [tteck/Proxmox#268] [General]
In your Debian 11 LXC I tried installing File Browser but get:

ERROR "1@64" Unknown failure occured.

I pretty much then followed the commands in the script manually and it all seemed to work! So not sure what is failing. 

Cheers.

**Comment by tteck** [comment]
```
  ______ _ _      ____                                  
 |  ____(_) |    |  _ \                                 
 | |__   _| | ___| |_) |_ __ _____  v3  _____  ___ _ __ 
 |  __| | | |/ _ \  _ <|  __/ _ \ \ /\ / / __|/ _ \  __|
 | |    | | |  __/ |_) | | | (_) \ V  V /\__ \  __/ |   
 |_|    |_|_|\___|____/|_|  \___/ \_/\_/ |___/\___|_|   

 ✓ Installed FileBrowser
 ✓ Created Service
 ✓ Completed Successfully!

FileBrowser should be reachable by going to the following URL.
         http://192.168.86.250 :8080 

root@debian:~# 
```
Can't replicate the issue. I'll keep an eye open. Thanks for the heads up.

**Comment by tteck** [comment]
Could have been a short github hiccup. Seems to be happening a lot lately. 🤷🏻‍♂️

**Comment by nigiriemoji** [comment]
I currently get this error but a different number

```
    _______ __     ____                                   
   / ____(_) /__  / __ )_________ _      __________  _____
  / /_  / / / _ \/ __  / ___/ __ \ | /| / / ___/ _ \/ ___/
 / __/ / / /  __/ /_/ / /  / /_/ / |/ |/ (__  )  __/ /    
/_/   /_/_/\___/_____/_/   \____/|__/|__/____/\___/_/     
                                                          
 - Installing FileBrowser...‼ ERROR 1@71 Unknown failure occured.
```

**Reply by tteck** [reply]
Try executing the command again.

**Reply by nigiriemoji** [reply]
> Try executing the command again.

Same thing. I recently had it working yesterday but I've done a bunch of adds and deletes of various LXCs from your page. Thanks so much for those, I'm slowly learning as I go but I think something I configured prior is interfering with this install perhaps. Anything I can run to troubleshoot this or view logs?

**Reply by tteck** [reply]
Try uninstalling, and then install. https://github.com/tteck/Proxmox/issues/1421#issuecomment-1546676016

**Reply by nigiriemoji** [reply]
> Try uninstalling, and then install. [#1421 (comment)](https://github.com/tteck/Proxmox/issues/1421#issuecomment-1546676016)

That worked, thanks lots!
